CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/hashmod/HashmodCSVImportFileParser.java

```diff
@@ -49,6 +51,9 @@ public List<String[]> parse(final HashmodInputSource input) throws IOException {
 
                 results.add(line);
             }
+        } catch (final Exception ex) {
```

Comment:
is there a more specific exception that can be caught here?

Reply:
Added catch for IOException and NullPointerException